lighthouse-core/audits/byte-efficiency/total-byte-weight.js

```diff
@@ -43,60 +43,61 @@ class TotalByteWeight extends ByteEfficiencyAudit {
    */
   static audit(artifacts) {
     const devtoolsLogs = artifacts.devtoolsLogs[ByteEfficiencyAudit.DEFAULT_PASS];
-    return artifacts.requestNetworkRecords(devtoolsLogs).then(networkRecords => {
-      return artifacts.requestNetworkThroughput(networkRecords).then(networkThroughput => {
-        let totalBytes = 0;
-        let results = [];
-        networkRecords.forEach(record => {
-          // exclude data URIs since their size is reflected in other resources
-          // exclude unfinished requests since they won't have transfer size information
-          if (record.scheme === 'data' || !record.finished) return;
+    return Promise.all([
+      artifacts.requestNetworkRecords(devtoolsLogs),
+      artifacts.requestNetworkThroughput(devtoolsLogs)
+    ]).then(([networkRecords, networkThroughput]) => {
+      let totalBytes = 0;
```

Comment:
the changes here (and in `critical-request-chains.js`) are just indentation (and, sadly, just reversing the indenting #2062 did :P)